npm/debugTools/dataplane/converter.go

```diff
@@ -0,0 +1,403 @@
+package dataplane
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/npm"
+	"github.com/Azure/azure-container-networking/npm/debugTools/pb"
+	"github.com/Azure/azure-container-networking/npm/util"
+	"google.golang.org/protobuf/encoding/protojson"
+	networkingv1 "k8s.io/api/networking/v1"
+)
+
+// Converter struct
+type Converter struct {
+	ListMap           map[string]string
+	SetMap            map[string]string
+	RequiredChainsMap map[string]bool
+	NPMCache          *NPMCache
+}
+
+// NPMCache struct
+type NPMCache struct {
+	Exec             interface{}
+	Nodename         string
+	NsMap            map[string]*npm.Namespace
+	PodMap           map[string]*npm.NpmPod
+	RawNpMap         map[string]*networkingv1.NetworkPolicy
+	ProcessedNpMap   map[string]*networkingv1.NetworkPolicy
+	TelemetryEnabled bool
+}
+
+// NpmCacheFromFile initialize NPM cache from file
+func (c *Converter) NpmCacheFromFile(npmCacheJSONFile string) error {
+	c.NPMCache = &NPMCache{}
+	// for dev
+	byteArray, err := ioutil.ReadFile(npmCacheJSONFile)
+	if err != nil {
+		return fmt.Errorf("error occurred during reading in file : %w", err)
+	}
+	err = json.Unmarshal(byteArray, c.NPMCache)
+	if err != nil {
+		return fmt.Errorf("error occurred during unmarshalling : %w", err)
+	}
+	return nil
+}
+
+// NpmCache initialize NPM cache from node
+func (c *Converter) NpmCache() error {
+	c.NPMCache = &NPMCache{}
+	req, err := http.NewRequestWithContext(
+		context.Background(),
+		http.MethodGet,
+		"http://localhost:10091/npm/v1/debug/manager",
```

Comment:
Instead of hard coding this, can you take this value from https://github.com/Azure/azure-container-networking/blob/a68b7fd5318d65378a46f4d2bc83d5492ef6b763/npm/http/api/api.go#L8